internal/multicluster/internal/types/partition_exported_services_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
We should add ACL checks for all these resources similar to this https://github.com/hashicorp/consul/blob/main/internal/catalog/internal/types/failover_policy_test.go#L668.